pkg/controller/controller_utils.go

```diff
@@ -342,6 +342,14 @@ func NewUIDTrackingControllerExpectations(ce ControllerExpectationsInterface) *U
 	return &UIDTrackingControllerExpectations{ControllerExpectationsInterface: ce, uidStore: cache.NewStore(UIDSetKeyFunc)}
 }
 
+// Reasons for pod events
+const (
+	FailedCreatePodReason     = "FailedCreate"
```

Comment:
Maybe? So far they have been plain strings so I don't think we should urge to make them part of the API.